controllers/articles.js

```diff
@@ -142,6 +150,38 @@ const articles = {
         });
       }
 
+      // tag registration
+      const oldTags = findArticle.tagList;
+      const newTags = req.body.tagList;
```

Comment:
@fridolinho Since we support request sent with **content-type** **application/json** and **application/x-www-from-url-encoded** we have to add some line of code to check the content type used to handle the tagList. When it is `application/json` tagList is an array and when it is `application/x-www-from-url-encoded` tagList is a string.
Check on line 44 of this file to see how to solve that.